### PR TITLE
Setup React Repo

### PR DESCRIPTION
**Pull request by calebchris000**
## The following requirements were met:

- First, make sure you have [Node.js](https://nodejs.org/en/) installed on your machine
- The easiest way to create a React application is using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)
- The application should run in the browser without errors

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @calebchris000, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have the Readme set up professionally, you can copy [this](https://github.com/microverseinc/readme-template) and domesticate it to fit your project. Also, remember to add the linters so that it will guide your code.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# INVALID REVIEW REQUEST

Hi @calebchris000, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have linter tests.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "eslint-config-next": "13.2.4",
+    "next": "13.2.4",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
```
Comment (didierganthier) [inline package.json]: Well done on this app so far 💯  but as the requirement says, the app should be created with `Create React App` as it is proven to be the best way to start [building a new single-page application](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app) in React. 🏗️

File: next.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+};
+
+module.exports = nextConfig;
```
Comment (didierganthier) [inline next.config.js]: You seem to have created a `Next` app 🤔 this shows great knowledge but this needs to be a `React` app created with `create-react-app`, you can create your app by running this:
```
npx create-react-app my-app
```
If you already created the repo on GitHub and you cloned it, you can run this:
```
npx create-react-app .
```
Which means "Create a React app in the current directory" 😉

NB: This supposes of course that you have `node`, `npm`, and `create-react-app` installed. 🤓

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi @calebchris000 ,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct Gitflow ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

**Review by DammyShittu — APPROVED** [review]
Hi @calebchris000,

## APPROVED 🟢

## Highlights
✔️ Use of create-react-app command

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**